issue_07/script.js

```diff
@@ -0,0 +1,32 @@
+const getData = new Promise((resolve) => {
+    const lists = [{
+        img: "1.png",
+        alt: "画像1",
+        text: "ブックマーク"
+    }, {
+        img: "2.png",
+        alt: "画像2",
+        text: "メッセージ"
+    }];
+    setTimeout(() => {
+        resolve(lists);
+    }, 3000);
+});
+const loadingImg = document.getElementById('js-loadingImg');
+const ul = document.getElementById('js-ul');
```

Comment:
nits
It is easier to read if it is not between a promise and .then.

Reply:
Thank you for pointing that out!
Fixed.
https://github.com/yoshikawa-akr/js-lessons/pull/7/commits/007737cb67f9b643213d4f1f908e6475c4825bd7
https://github.com/yoshikawa-akr/js-lessons/pull/7/commits/01581850e7e22af0dd51f2a8dfc6407c5203c8f7